addons/base_import/models/base_import.py

```diff
@@ -600,6 +601,16 @@ def _parse_float_from_data(self, data, index, name, options):
                 raise ValueError(_("Column %s contains incorrect values (value: %s)" % (name, old_value)))
 
     @api.multi
+    def binary2base64(self, url):
```

Comment:
This method has no reason to be public, or to have a decorator. It is also horribly named, we're downloading to base64 not converting "binary" to it. I'd even question the point of having a separate method for that at this point.